buildmaster/H1_1JET_319GEV_290PB-1_DIF/metadata.yaml

```
@@ -0,0 +1,120 @@
+setname: 'H1_1JET_319GEV_290PB-1_DIF'
+
+nnpdf_metadata:
+  nnpdf31_process: "1JET"
+  experiment: "H1"
+
+arXiv:
+  url: "https://arxiv.org/abs/1611.03421"
+iNSPIRE:
+  url: "https://inspirehep.net/literature/1496981"
+hepdata:
+  url: "https://www.hepdata.net/record/ins1496981"
+  version: 1
+
+version: 1
+version_comment: "NA"
+
+implemented_observables:
+
+  - observable: {description: "double differential cross section w.r.t. q^2 and pT", label: '$\frac{d^2\sigma}{dq^{2}dpT}$', units: "$pb/GeV^3$"}
+    observable_name: PTQ2
+    process_type: DIS_JET
+    ndata: 48
+    tables: [1, 2, 3, 4, 5, 6, 7, 8]
+    kinematics:
+      variables:
+        pT: {description: "mean transverse momentum", label: '$pT$', units: "$GeV$"}
+        Q2: {description: "momentum transferred squared", label: '$q^2$', units: "$GeV^2$"}
+        sqrt_s: {description: "center of mass energy", label: '$\sqrt{s}$', units: "$GeV$"}
+      file: kinematics.yaml
+    data_central: data.yaml
+    data_uncertainties:
+      - uncertainties.yaml
+    kinematic_coverage: [pT, Q2, sqrt_s]
+    plotting:
+      kinematics_override: identity
+      dataset_label: '$H1 Jet 290 pb^{-1} distribution: \frac{d^2\sigma}{dq^{2}dpT}$'
+      plot_x: pT
+      x_scale: log
+      figure_by: 
+        - Q2
+    theory:
```

```suggestion
    theory:
        shifts:
            H1_1JET_319GEV_290PB-1_DIF_PTQ2_BIN1: 6
```

This tells validphys to read the fktables such that the first point in the fktables is actually point 6.

This flag used to be called `apfelcomb::shifts` because it was not needed for any of the pineappl tables but since it clearly will be (particularly if we are not the ones generating the grids) I'll change the name in the reader to simply ``shifts``.

I'm going to update the reader to deal with this.

---

done